lib/surgex/date_time/date_time.ex

```diff
@@ -11,12 +11,17 @@ if Code.ensure_loaded?(Timex) do
       iex> Surgex.DateTime.date_and_offset_to_datetime(~D{2021-10-07}, 5400)
       {:ok, ~U[2021-10-07 01:30:00Z]}
     """
-    @spec date_and_offset_to_datetime(Date.t(), integer(), String.t()) ::
+    @spec date_and_offset_to_datetime(Date.t(), integer(), String.t(), atom()) ::
```

Comment:
If I'm not mistaken, this only makes sense for `:before` and `:after` atoms, but I had to go to `Timex` documentation to check it. Maybe the type spec could hint that?